### DISCUSSION TITLE
Make a cell span multiple cells in a Grid?

### DISCUSSION THREAD

**Discussion by SolenOchHavet** [emilk/egui#1511] [Q&A] [answered]
Hi!

I have looked through all the amazing examples but could not find any hint on if it's possible to make a cell in a Grid span over several cells (horizontal in my case)? I looked at the documentation as well but couldn't find anything. I can work around the issue I think, just wanted to double check if I missed it :)

Thanks in advance,
Andreas

**Comment by emilk** [comment, marked as answer]
No you can't, but you can with the new `egui_extras::Strip` coming in 0.18.0: https://github.com/emilk/egui/pull/963
